resizescript.py

```diff
@@ -0,0 +1,29 @@
+from PIL import Image
+import os,sys
+
+path = "/Users/ezraschwartz/Documents/thesis/images/"
```

Comment:
Don't hardcode these paths. Make them arguments to the script.

Reply:
Fixed this